plugins/basestate/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+project(basestate)
```

Comment:
```suggestion
project(basestate)
```

I think we can remove this.

Reply:
We can remove this since it is not referenced anywhere right?